docs/source/gbd2019_models/risk_effects/ldl/index.rst

```diff
@@ -105,76 +93,80 @@ Vivarium Modeling Strategy
+     - False
      -
+   * - Age group start
+     - 10
+     - [25, 29)
+   * - Age group end
+     - 235
+     - [95, 125 years)
 
 Risk Outcome Pair #1
 ++++++++++++++++++++
 
-.. todo::
-
-	Replace "Risk Outcome Pair #1" with the name of an affected entity for which a modeling strategy will be detailed. For additional risk outcome pairs, copy this section as many times as necessary and update the titles accordingly.
-
-.. todo::
+:ref:`See ischemic heart disease documentation <2019_cause_ihd>`
 
-  Link to existing cause model document or other documentation of the outcome in the risk outcome pair.
+The relative risks apply to the incidence rates of acute myocardial infarction and stable angina. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r367,c493`\) * RR\ :sub:`r367`\). The association was evaluated at the cause level, but the associations should be applied to the incidence rates for both nonfatal components of ischemic heart disease. 
 
-.. todo::
+PAFs and relative risks can be pulled using the following code: 
 
-  Describe which entitity the relative risks apply to (incidence rate, prevalence, excess mortality rate, etc.) and *how* to apply them (e.g. :code:`affected_measure * (1 - PAF) * RR`).
+rrs = get_draws(gbd_id_type='rei_id', gbd_id=367, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 
 
-  Be sure to specify the exact PAF that should be used in the above equation and either how to calculate it (see the `Population Attributable Fraction` section of the :ref:`Modeling Risk Factors <models_risk_factors>` document) or pull it (:code:`vivarium_inputs.interface.get_measure(risk_factor.{risk_name}, 'population_attributable_fraction')`, noting which affected entity and measure should be used)
+pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[367, 493], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
 
-.. todo::
 
-  Complete the following table to list the relative risks for each risk exposure category on the outcome. Note that if there are many exposure categories, another format may be preferable.
+Risk Outcome Pair #2
+++++++++++++++++++++
 
-  Relative risks for a risk factor may be pulled from GBD at the draw-level using :code:`vivarium_inputs.interface.get_measure(risk_factor.{risk_name}, 'relative_risk')`. You can then calculate the mean value as well as 2.5th, and 97.5th percentiles across draws.
+:ref:`See ischemic stroke documentation <2019_cause_ischemic_stroke>`
 
-  The relative risks in the table below should be included for easy reference and should match the relative risks pulled from GBD using the above code. In this case, update the :code:`Note` below to include the appropriate :code:`{risk_name}`.
+The relative risks apply to the incidence rates of acute ischemic stroke. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r367,c495`\) * RR\ :sub:`r367`\). 
```

Comment:
As above, I was expecting the exposure and TMREL to be included in this formula, e.g.

incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}

(But now that I think about it, I think there might be some additional transformation, e.g. dividing max(LDL-C_i - TMREL, 0) by some LDL RR scale like 5.0.

Reply:
incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}
^+: indicates max(LDL-C<TMREL, 0)
for SBP, (max(SBP_i - TMREL, 0))/10 bc RR is for a 10 unit increase in Hg

Reply:
this is the formula for using a log-linear risk; will need to be updated to address the GBD 2020 approach with splines.